### ISSUE TITLE
WWE 2K BATTLEGROUNDS (1142100)

### ISSUE THREAD

**Issue by NoXPhasma** [ValveSoftware/Proton#4743]
# Compatibility Report
- Name of the game with compatibility issues: WWE 2K BATTLEGROUNDS
- Steam AppID of the game: 1142100

## System Information
- GPU: GTX 1080
- Driver/LLVM version: 465.19.01
- Kernel version: 5.11
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/NoXPhasma/0a9209bd6b123723197b5652652ffa8e
- Proton version: 6.3-2

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

[steam-1142100.log](https://github.com/ValveSoftware/Proton/files/6278300/steam-1142100.log)

## Symptoms 

Game starts up, shows test screen for the Media Foundation videos and then freezes before showing the main menu.

## Reproduction

- Set Proton 6.3-2 as compatibility tool
- Launch the game

This is a regression with 6.3-2, the game works fine with Proton 5.13-6, beside videos not being shown due to Media Foundation.
[Proton 5.13-6 steam-1142100.log](https://github.com/ValveSoftware/Proton/files/6278346/steam-1142100.log)

**Comment by kisak-valve** [comment]
Hello @NoXPhasma, can you test if setting the game's launch options to `WINEDLLOVERRIDES="mfplay.dll=d" %command%` works around the game freeze?

**Comment by NoXPhasma** [comment]
Yes, that fixed the freeze. And obviously stopped playing the test screen videos.

[steam-1142100.log](https://github.com/ValveSoftware/Proton/files/6278648/steam-1142100.log)
